.github/workflows/fastapi-test.yaml

```
@@ -35,11 +35,19 @@ jobs:
-          ./scripts/setup
+          ./backend/scripts/setup
+      # - name: Change to backend directory
+      #   run: |
+      #     cd backend
+      #     echo $(pwd)
+      #     ls -a
       - name: black check
         run: black --diff --check $(git ls-files '*.py')
       - name: isort check
-        run: isort --diff --check --profile black $(git ls-files '*.py')
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2046:warning:1:6: Quote this to prevent word splitting

(shellcheck)